README.md

```diff
@@ -129,6 +129,7 @@ jobs:
 | `coverity_passphrase`        | Passphrase for Coverity    | Mandatory     |
 | `coverity_project_name`        | Project name in Coverity. <br> Many customers prefer to set their Coverity project and stream names to match the GitHub repository name  </br>                     | Mandatory     |
 | `coverity_stream_name`        | Stream name in Coverity   | Mandatory     |
+| `bridge_coverity_local`                | If enabled, Synopsys Action will install the ‘Coverity Thick Client’ on the local system in order to execute the scan. 'coverity_install_directory' can also be given to keep the files for future scans.   | Optional     |
```

Comment:
@lsynopsys - i have updated this readme MR. please verify and make changes in code MR accordingly.

Reply:
done